SA3/qa_web/views.py

```diff
@@ -84,19 +84,21 @@ def answers(request, id_):
         if form.is_valid(): 
             Answers.objects.create(content=request.POST['content'], owner=request.user, question=q)
     elif request.method == 'POST' and 'deselect' in request.POST:  #Update's database when somebody deselects best answer.
-        updateAnswer = Answers.objects.filter(question=q, correct_answer=True).last()
+        updateAnswer = Answers.objects.get(question=q, correct_answer=True)
         updateAnswer.correct_answer = False;
         updateAnswer.save();
     else: #Update's database when somebody selects a best answer.
```

Comment:
Make sure to specify this last else as elif request.method == 'POST'. GET requests would still be able to execute and what not.

Reply:
Apologies for overlooking such a trivial error.